src/main/java/org/opentripplanner/middleware/controllers/api/UserController.java

```diff
@@ -49,4 +81,18 @@ boolean preDeleteHook(OtpUser user, Request req) {
         }
         return true;
     }
+
+    /**
+     * HTTP endpoint to get the {@link OtpUser} entity, if it exists, from an {@link Auth0UserProfile} attribute
+     * available from a {@link Request} (this is the case for '/api/secure/' endpoints).
+     */
+    private OtpUser getUserFromRequest(Request req, Response res) {
+        Auth0UserProfile profile = Auth0Connection.getUserFromRequest(req);
+        OtpUser user = profile.otpUser;
```

Comment:
Does this need to consider an apiUser and adminUser users?

Reply:
`UserController` is for `OtpUser`s.

Reply:
Of course. Thanks for clarifying.